.travis.yml

```diff
@@ -1,3 +1,10 @@
-language: node_js
-node_js:
-  - "node"
+matrix:
+    include:
+      - language: java
+        before_install:
+          - wget https://dl.bintray.com/spdx/spdx-tools/org/spdx/spdx-tools/2.1.8/spdx-tools-2.1.8-jar-with-dependencies.jar
+        script: 
+          - 'if [[ "$TRAVIS_PULL_REQUEST" = "false" && "$TRAVIS_BRANCH" = "master" ]]; then bash ./.travis/publishLicenseList.sh; else bash ./.travis/testBuildLicenseList.sh; fi'
```

Comment:
Both scripts test, the publish scripts test AND publish.  Since these are long running scripts, I don't want to duplicate the testing that is done as part of publishing.  I'll rename the publish script to make this clearer.

Pull requests do matter in terms of security since it may be on an external repo.

I'll re-order the test on the master branch and will drop `bash`

Reply:
> Pull requests do matter in terms of security since it may be on an external repo.

How does checking the branch name and or PR-ness help address the external-repo case?  I'd expect the guard for that being “folks with push rights to license-list-data don't abuse their power”, and not something that happens in the script here.